fireant/slicer/queries/builder.py

```diff
@@ -228,7 +221,7 @@ def __init__(self, slicer, dimension):
         self._dimensions.append(dimension)
 
     @property
-    def query(self):
+    def queries(self):
         """
         Serialize a pypika/SQL query from the slicer builder query.
```

Comment:
Imo all methods that got renamed from query to queries should have their docstrings updated, to reflect the fact that them now return a list of serialized queries.

Reply:
yes, you're right. I still needed to add/update comments in a couple places. This is done now.